ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import Box from '../../../ui/box';
+import Typography from '../../../ui/typography';
+import CheckBox from '../../../ui/check-box';
+import I18nValue from '../../../ui/i18n-value';
+import {
+  COLORS,
+  DISPLAY,
+  FLEX_DIRECTION,
+  TYPOGRAPHY,
+} from '../../../../helpers/constants/design-system';
+import { getAdvancedGasFeeValues } from '../../../../selectors';
+import { setAdvancedGasFee } from '../../../../store/actions';
+
+import { useAdvancedGasFeePopoverContext } from '../context';
+
+const AdvancedGasFeeDefaults = () => {
+  const dispatch = useDispatch();
+
+  const {
+    baseFeeMultiplier,
+    maxPriorityFeePerGas,
+  } = useAdvancedGasFeePopoverContext();
+  const advancedGasFeeValues = useSelector(getAdvancedGasFeeValues);
+
+  const updateDefaultSettings = (value) => {
```

Comment:
I'm wondering if it makes more sense to break this function into two functions? It seems that this does two distinct things: update the default settings (based on whatever the current value of `baseFeeMultiplier` and `maxPriorityFeePerGas` is) and clear the default settings.

Reply:
The function basically just sets and clears the `advancedGasFee` object when the checkbox is checked and unchecked.